pkg/kubectl/explain/field_lookup_test.go

```diff
@@ -69,9 +69,9 @@ func TestFindField(t *testing.T) {
 			gotPath = path.GetPath().String()
 		}
 
-		if gotErr != test.err && gotPath != test.expectedPath {
```

Comment:
My code is definitely wrong! Thanks for noticing :-)

Reply:
:-) you are amazing!!!! it's just a small thing.